File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.js
function solution(binomial) {
    const arr = binomial.split(' ');
    
    if (arr[1] === '+') {
        return arr[0] * 1 + arr[2] * 1;
    } else if (arr[1] === '-') {
        return arr[0] * 1 - arr[2] * 1;
    } else {
        return arr[0] * 1 * arr[2] * 1;
    }
}

File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.java
class Solution {
    public int solution(String binomial) {
        String[] split = binomial.split(" ");
        
        if (split[1].equals("+")) {
            return Integer.parseInt(split[0]) + Integer.parseInt(split[2]);
        }
        if (split[1].equals("-")) {
            return Integer.parseInt(split[0]) - Integer.parseInt(split[2]);
        }
        
        return Integer.parseInt(split[0]) * Integer.parseInt(split[2]);
    }
}